models/public/cocosnet/cocosnet.md

```diff
@@ -0,0 +1,93 @@
+# cocosnet
+
+## Use Case and High-Level Description
+
+Cross-domain correspondence network is a exemplar-based image translation model, consisting of correspondence and translation parts. Model was pre-trained on ADE20k dataset.
+For details see [paper](https://arxiv.org/pdf/2004.05571) and [repository](https://github.com/microsoft/CoCosNet).
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Image_translation                         |
+| GFLOPs                          | 1080.7032                                 |
+| MParams                         | 167.9141                                  |
+| Source framework                | PyTorch\*                                 |
+
+## Accuracy
+
+Metrics was calculated between generated images by model and real validation images from ADE20k dataset.
+For some GAN metrics (IS and FID) you need to use classification model as verification network.
+In our case it is [Inception-V3](../googlenet-v3/googlenet-v3.md) model.
+
+| Metric | Original model | Converted model |
+| ------ | -------------- | --------------- |
+| PSNR   | 12.99dB        | 12.93dB         |
+| SSIM   | 0.34           | 0.34            |
+| IS     | 13.34          | 13.35           |
+| FID    | 33.27          | 33.14           |
+
+### Inputs
+
+1. name: "input_seg_map", shape: [1x151x256x256] - Input semantic segmentation mask (one-hot label map) in the format [BxCxHxW],
+   where:
+    - B - batch size
+    - C - number of classes (151 for ADE20k)
+    - H - mask height
+    - W - mask width
+
+2. name: "ref_image", shape: [1x3x256x256] - An reference image (exemplar) in the format [BxCxHxW],
+   where:
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+    Expected color order is BGR.
+
+3. name: "ref_seg_map", shape: [1x151x256x256] - A mask (one-hot label map) for reference image in the format [BxCxHxW],
+   where:
+    - B - batch size
+    - C - number of classes (151 for ADE20k)
+    - H - mask height
+    - W - mask width
+
+### Output
+
+Image, name: "exemplar_based_output", shape: [1x3x256x256] - A result (generated) image based on exemplar in the format [BxCxHxW],
+   where:
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+    Output color order is RGB.
+
+## Legal Information
+
+The original model is distributed under the following
+[license](https://github.com/microsoft/CoCosNet/blob/master/LICENSE):
```

Comment:
You should also list the license of the Synchronized-BatchNorm-PyTorch project that's used as the dependency.